src/XMakeCommandLine/UnitTests/project.json

```diff
@@ -7,28 +7,25 @@
   },
   "frameworks": {
     "net451": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
-      }
+      "dependencies": {}
     },
     "net46": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
```

Comment:
I'm worried about this since I'm planning to turn full-framework builds back on for this branch, and they'll need this.  We can punt it for a while, though.